### ISSUE TITLE
Instalation script is broken

### ISSUE THREAD

**Issue by kyob** [theInfectedDrake/TIDoS-Framework#23]
**Installation:**
```bash
git clone https://github.com/theinfecteddrake/tidos-framework.git
cd tidos-framework
chmod +x install
./install
```

**Logs:**
```bash
Checking for pre-installations...
Removing any trace of pre-installations...
removed '/opt/tidos-framework/README.md'
removed '/opt/tidos-framework/files/brute-db/smtp/smtp_defuser.lst'
removed '/opt/tidos-framework/files/brute-db/smtp/smtp_defpass.lst'
[...]
removed '/opt/tidos-framework/setup/__init__.py'
removed directory '/opt/tidos-framework/setup'
removed directory '/opt/tidos-framework'
Setting necessary permissions...
sh: 0: getcwd() failed: No such file or directory
chmod: cannot access 'dependencies': No such file or directory
failed to change mode of 'dependencies' from 0000 (---------) to 0000 (---------)
Processing dependencies...
sh: 0: getcwd() failed: No such file or directory
shell-init: error retrieving current directory: getcwd: cannot access parent directories: No such file or directory
bash: dependencies: No such file or directory
Creating directories...
sh: 0: getcwd() failed: No such file or directory
mkdir: created directory '/opt/tidos/'
Copying new files...
sh: 0: getcwd() failed: No such file or directory
cp: cannot stat '*': No such file or directory
Creating shortcuts...
sh: 0: getcwd() failed: No such file or directory
cp: cannot stat 'runon.sh': No such file or directory
Giving priviledges...
sh: 0: getcwd() failed: No such file or directory
chmod: cannot access '/opt/tidos/*': No such file or directory
sh: 0: getcwd() failed: No such file or directory
chmod: cannot access '/usr/bin/tidos': No such file or directory
failed to change mode of '/usr/bin/tidos' from 0000 (---------) to 0000 (---------)
```

After run **install** then directory **tidos-framework** is removed and directory **tidos** is empty.

**Comment by gbiagomba** [comment]
I tend to find it that running "python2 setup.py install" is the most successful approach. In the past on other installs or scripts things have gone haywire when I try to do ./script.py but then when I use the command mentioned earlier the issue(s) go away. 

Fruit for thought

**Comment by theInfectedDrake** [comment]
I am not able to reproduce the error. However there were some major code changes past the new release of v1.6.1 which I published just now. Try getting a fresh clone and tell me if the error exists.

I guess all of these are happening in my attempt to fix #17 in 63ca594. I had to change the installation directory to `/opt/tidos/` from `/opt/tidos-framework/`. That is why all of this is messing up. I would recommend a full cleanup of the `/opt/` folder, void of all previous installations of TIDoS and get a fresh clone. it should work fine af.

I am closing this for now. let me know if you get a similar error. i would be happy to re-open this again
